### PR TITLE
chore(tick): Tick as number instead of BigNumber

### PR DESCRIPTION
**Pull request by lnist**

### CODE REVIEW

File: src/util/coreCalculations/Constants.ts
Comment (espendk) [inline src/util/coreCalculations/Constants.ts:1]: I think we should keep the mechanically translated core calculations files as is. And then map to/from in the code that interact with them.

Comment (espendk) [reply src/util/coreCalculations/Constants.ts:1]: Wrt. MAX_TICK and MIN_TICK which are referenced outside of Density and TickPriceHelper (the wrappers around core calculations): We can export them as `number` in TickPriceHelper instead. This way only imports need to be updated.

**Review by espendk — CHANGES_REQUESTED** [review]
lgtm. Don't think we should change the core calculations code though, see the comments on Constants.ts

File: src/util/coreCalculations/Constants.ts [hunk 1]
```diff
@@ -65,8 +65,8 @@ export const MIN_BIN = BigNumber.from("-1048576");
 export const MAX_BIN = BigNumber.from("1048575");
 
 /* The tick range is the largest such that the mantissa of `1.0001^MAX_TICK` fits on 128 bits (and thus can be multiplied by volumes). */
-export const MIN_TICK = BigNumber.from("-887272");
-export const MAX_TICK = BigNumber.from("887272");
+export const MIN_TICK = BigNumber.from(-887272);
+export const MAX_TICK = BigNumber.from(887272);
```
Comment (espendk) [inline src/util/coreCalculations/Constants.ts:69]:
```suggestion
export const MIN_TICK = BigNumber.from("-887272");
export const MAX_TICK = BigNumber.from("887272");
```

File: src/util/coreCalculations/TickLib.ts [hunk 2]
```diff
@@ -117,7 +117,7 @@ export type Tick = int;
 // library TickLib {
 
   export function inRange(tick: Tick): boolean {
-    return tick >= MIN_TICK && tick <= MAX_TICK;
+    return tick.gte(MIN_TICK) && tick.lte(MAX_TICK);
```
Comment (espendk) [inline src/util/coreCalculations/TickLib.ts:120]: good catch!

File: src/util/coreCalculations/Constants.ts
Comment (lnist) [reply src/util/coreCalculations/Constants.ts:1]: Very few places used them, so I decided to just do a toNumber

**Review by espendk — CHANGES_REQUESTED** [review]
small nit on the Constants change revert ;-)

**Review by espendk — APPROVED** [review]

**Pull request merged**